appveyor.yml

```diff
@@ -0,0 +1,36 @@
+version: 1.0.{build}
+clone_depth: 1
+
+configuration:
+    - Debug
+    - Release
+
+os: Visual Studio 2015
+image: Visual Studio 2015
+
+platform:
+    - x64
+
+environment:
+    BOOST_ROOT: C:\Libraries\boost_1_59_0\boost
```

Comment:
That's ok though, it's the path to the appveyor provided install, which is documented and all.